tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py

```diff
@@ -20,7 +19,7 @@
 
 
 @tier1
-@pytest.mark.post_deployment
+@pytest.mark.first
```

Comment:
have you checked that with this pytest ordering first marker this test is not running before 'test_deploymet()' from https://github.com/red-hat-storage/ocs-ci/blob/master/tests/ecosystem/deployment/test_deployment.py?

Reply:
This is marked as tier1 test so we do not trigger it with deployment but with tier1 suite

Reply:
thanks!